submissions/opeyemiPeter/DOM/script.js

```diff
@@ -0,0 +1,26 @@
+// Newsletter task
+// add the html 
+
+const submitForm = document.getElementById('newsletter');
+const email = document.getElementById('email');
+const emailRegex = /^\w+[\w-\.]*\@\w+((-\w+)|(\w*))\.[a-z]{2,3}$/;
+
+
+    function validateForm(e){
+        e.preventDefault();
```

Comment:
Put the `e.preventDefault();` inside the block that return false. Else, even when everything passes, it will prevent the form from submitting

Reply:
i will fix that now